pkg/bindings/cre_common.go

```diff
@@ -0,0 +1,6 @@
+package bindings
+
+// This function is not EVM specific, it's generic and should be provided by CRE
```

Comment:
Can you create a ticket to track pending stuff like plugging in the actual function for generating a report once that's done?

Reply:
https://smartcontract-it.atlassian.net/browse/PLEX-1555